custom_components/myskoda/coordinator.py

```diff
@@ -36,63 +58,93 @@ class MySkodaDataUpdateCoordinator(DataUpdateCoordinator[State]):
     myskoda: MySkoda
     config: ConfigEntry
     data: State
-
-    def __init__(self, hass: HomeAssistant, config: ConfigEntry) -> None:
+    update_driving_range: RefreshFunction
+    update_charging: RefreshFunction
+    update_air_conditioning: RefreshFunction
+    update_vehicle: RefreshFunction
+    vin: str
+    hass: HomeAssistant
+
```

Comment:
Only define the types on class level if we can't infer the types

Reply:
Implemented in 3cac6d4 except for `data: State` as discussed in discord